sdl_viewer/src/main.rs

```diff
@@ -302,10 +308,22 @@ fn main() {
                     .help("Input directory of the octree directory to serve.")
                     .index(1)
                     .required(true),
+                clap::Arg::with_name("cache_size")
+                    .help("Maximum number of octree nodes to store in GPU memory")
+                    .required(false)
             ]
         )
         .get_matches();
 
+    // cache size
+    let mut cache_size = 10000;
```

Comment:
`let mut cache_size = min(5000, matches.value_of("cache_size").unwrap_or("10000").map(|s| s.parse().unwrap()))` should work.